bitnami/grafana/README.md

```diff
@@ -0,0 +1,180 @@
+
+```console
+$ helm delete my-release
+```
+
+The command removes all the Kubernetes components associated with the chart and deletes the release. Use the option `--purge` to delete all persistent volumes too.
+
+## Configuration
+
+The following tables lists the configurable parameters of the grafana chart and their default values.
+
+| Parameter                                   | Description                                                                                 | Default                                                 |
+| ------------------------------------------- | ------------------------------------------------------------------------------------------- | ------------------------------------------------------- |
+| `global.imageRegistry`                      | Global Docker image registry                                                                | `nil`                                                   |
+| `global.imagePullSecrets`                   | Global Docker registry secret names as an array                                             | `[]` (does not add image pull secrets to deployed pods) |
+| `image.registry`                            | Grafana image registry                                                                      | `docker.io`                                             |
+| `image.repository`                          | Grafana Image name                                                                          | `bitnami/grafana`                                       |
+| `image.tag`                                 | Grafana Image tag                                                                           | `{TAG_NAME}`                                            |
+| `image.pullPolicy`                          | Grafana image pull policy                                                                   | `IfNotPresent`                                          |
+| `image.pullSecrets`                         | Specify docker-registry secret names as an array                                            | `[]` (does not add image pull secrets to deployed pods) |
+| `nameOverride`                              | String to partially override grafana.fullname template with a string (will prepend the release name) | `nil`                                          |
+| `fullnameOverride`                          | String to fully override grafana.fullname template with a string                            | `nil`                                                   |
+|`replicaCount`                               | Number of replicas of the Grafana Pod                                                       | `1`                                                     |
+| `updateStrategy.type`                       | Update strategy for deployment                                                              | `RollingUpdate`                                         |
```

Comment:
You're using the whole `updateStrategy` object in the `deployment.yaml`:
```yaml
  {{- if .Values.updateStrategy }}
  strategy: {{ toYaml .Values.updateStrategy | nindent 4 }}
  {{- end }}
```
Therefore, I'll document this parameter as follows:
```
| `updateStrategy` | Update strategy for Grafana deployment | `{type: "RollingUpdate"}`    |
```